flake.nix

```
@@ -115,6 +115,12 @@
             export XDG_RUNTIME_DIR="$(mktemp -d)"
           '';
 
+          checkFlags = [
+            # These tests require the ability to access a "valid EGL Display", but that won't work
+            # inside the Nix sandbox
+            "--skip=tests::animations
```

```suggestion
            "--skip=tests::animations"
```

Oops that's what I get for applying from mobile.